coprarna_aux/prepare_intarna_out.pl

```diff
@@ -130,12 +133,15 @@
             }
         }
         my $ncrnafilename = $ncrnaarray[0]; 
-        $ncrnafilename = $ncrnafilename . ".fa";
-        @ncrnaarray = ();
+		my $ncrnafilename1 = $ncrnaarray[0];
+			$ncrnafilename = $ncrnafilename . ".fa";
+			@ncrnaarray = ();
+		system("mkdir $ncrnafilename1"); # edit jens
         my $intarnaout = $_ . ".intarna.csv"; ## edit 2.0.5.1 // added .csv
             $pm->start and next;            
-            system("IntaRNA --tAccW $winsize --tAccL $maxbpdist --outNumber 2 --target $_ --query $ncrnafilename --outCsvCols 'id1,id2,seq1,seq2,subseq1,subseq2,subseqDP,subseqDB,start1,end1,start2,end2,hybridDP,hybridDB,E,ED1,ED2,Pu1,Pu2,E_init,E_loops,E_dangleL,E_dangleR,E_endL,E_endR,seedStart1,seedEnd1,seedStart2,seedEnd2,seedE,seedED1,seedED2,seedPu1,seedPu2,E_norm' --outMode=C --out $intarnaout") unless (-e $intarnaout); ## edit 2.0.4 added -s 1 for consensus prediction // ## edit 2.0.5 changed to IntaRNA2 // ## edit 2.0.5.1 changed to IntaRNA 2.0 csv output
-            print("IntaRNA --tAccW $winsize --tAccL $maxbpdist --outNumber 2 --target $_ --query $ncrnafilename --outCsvCols 'id1,id2,seq1,seq2,subseq1,subseq2,subseqDP,subseqDB,start1,end1,start2,end2,hybridDP,hybridDB,E,ED1,ED2,Pu1,Pu2,E_init,E_loops,E_dangleL,E_dangleR,E_endL,E_endR,seedStart1,seedEnd1,seedStart2,seedEnd2,seedE,seedED1,seedED2,seedPu1,seedPu2,E_norm' --outMode=C --out $intarnaout\n") if ($verbose);
+            #system("/home/jens/For_CopraRNA2.0/IntaRNA-dev190704 --model=X --tAccW $winsize --tAccL $maxbpdist  --target $_ --query $ncrnafilename --outCsvCols 'id1,id2,seq1,seq2,subseq1,subseq2,subseqDP,subseqDB,start1,end1,start2,end2,hybridDP,hybridDB,E,ED1,ED2,Pu1,Pu2,E_init,E_loops,E_dangleL,E_dangleR,E_endL,E_endR,seedStart1,seedEnd1,seedStart2,seedEnd2,seedE,seedED1,seedED2,seedPu1,seedPu2,E_norm' --outMode=C --out $intarnaout --out=SpotProb:./$ncrnafilename1/probs.csv") unless (-e $intarnaout); ## edit 2.0.4 added -s 1 for consensus prediction // ## edit 2.0.5 changed to IntaRNA2 // ## edit 2.0.5.1 changed to IntaRNA 2.0 csv output
+            system("IntaRNA --outOverlap=Q --tAccW $winsize --tAccL $maxbpdist  --target $_ --query $ncrnafilename --outCsvCols 'id1,id2,seq1,seq2,subseq1,subseq2,subseqDP,subseqDB,start1,end1,start2,end2,hybridDP,hybridDB,E,ED1,ED2,Pu1,Pu2,E_init,E_loops,E_dangleL,E_dangleR,E_endL,E_endR,seedStart1,seedEnd1,seedStart2,seedEnd2,seedE,seedED1,seedED2,seedPu1,seedPu2,E_norm' --outMode=C --out $intarnaout --out=SpotProb:./$ncrnafilename1/probs.csv") unless (-e $intarnaout);
```

Comment:
@JensGeorg brauchst du die ganzen CSV columns im output überhaupt? wenn nicht, kürz mal `--outcsvcols` auf das notwendige ein...

Reply:
Also ich brauche (wharscheinlich) nur folgende Columns: --outCsvCols 'id1,id2,seq1,seq2,start1,end1,start2,end2,hybridDP,E,seedStart1,seedEnd1,seedStart2,seedEnd2'

Müßte ich dann halt nochmal testen. Ich weiß aber nicht welche Columns in Patricks Scripten gebraucht werden.

Reply:
habs erstmal geschrumpft und wir schaun mal obs crashed.. ;)